src/ssda/util/salt_observation.py

```diff
@@ -55,7 +55,10 @@ def create_reduced_path(path: Path) -> Optional[Path]:
 
             for _path in reduced_paths:
                 if _path.name.endswith(path.name):
-                    _reduced_path = _path
+                    if not _reduced_path:
+                        _reduced_path = _path
+                    if len(_path.name) > len(_reduced_path.name):
```

Comment:
Can be slightly simplified by using only one if statement with the condition `_path.name is None or len(_path.name) > len(_reduced_path.name)`.